v2/api-validator/src/config/index.ts

```diff
@@ -81,16 +85,24 @@ const config = convict({
               "testAsset": false
             },
             "transferMethod": "PublicBlockchain"
-          },
+          }
+        ]`,
+        env: 'COLLATERAL_WIHTDRAWAL_ADDRESSES',
+      },
+    },
+    deposit: {
+      assets: {
+        format: Array<AssetReference>,
+        default: `[
```

Comment:
A string containing a JSON?! All the purpose of this thing is to have a predictable structure and ability to configure this easily from the outside. Check, for example, how `withdrawal.iban` is defined and do the same.